### PR TITLE
enh(phpstan) :  update composer.json and composer.lock

### PR DESCRIPTION
**Pull request by AssanDiomande**
## Description

**Fixes** # MON-11920 (partially)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

* Use the command composer install
* Use the command composer show | grep phpstan
* check if the version of phpstan is 1.2
* check if the version of phpstan-beberlei-assert is 1.0.0

### CODE REVIEW

**Review by callapa — CHANGES_REQUESTED** [review]
Attention, you have downgraded the version of the library monolog/monolog.

**Comment by AssanDiomande** [comment]
> Attention, you have downgraded the version of the library monolog/monolog.

I compared the version between this branch and develop and I got the same version (1.26.1)

File: composer.lock [hunk 1]
```diff
@@ -4,20 +4,20 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "27c13592e891e1072575908e98dc4454",
+    "content-hash": "74ec0c199110e2586a3d4ff7b2b9c7ea",
     "packages": [
         {
             "name": "beberlei/assert",
-            "version": "v3.3.2",
+            "version": "v3.3.1",
```
Comment (callapa) [inline composer.lock]: Please do not downgrade the version.

**Review by callapa — CHANGES_REQUESTED** [review]

**Review by kduret — COMMENTED** [review]
Please set this PR as draft until it is not ready for review

**Pull request closed without merging**